cloudsensei/agg_lambda/lambda_function.py

```diff
@@ -22,7 +22,7 @@ def lambda_handler(event, context):
     message = "Something went wrong check your es_data"
     es_operations = ESOperations()
     email_body = ""
-    subject = "Daily Report of long running instances in Perf&Scale AWS Accounts"
```

Comment:
Weekly Cloud Report: Long-running instances in the Perf&Scale AWS Accounts?

Reply:
Updated.